network-protocols.md

```diff
@@ -18,7 +18,7 @@ Note: Implementations should limit the maximum number of bytes read during the `
 
 # Hello Handshake
 
-> The Hello protocol is used when two filecoin nodes initially connect to eachother in order to determine information about the other node.
```

Comment:
Hello handshake is defined twice, could you clean that up while you are in here?